pkg/proxy/iptables/proxier_test.go

```diff
@@ -1268,11 +1268,8 @@ func Test_accumulateEndpointsMap(t *testing.T) {
 				&endpointsInfo{"1.1.1.1:11", false},
 			},
 		},
-		expectedNew: map[proxy.ServicePortName][]*endpointsInfo{},
-		expectedStale: []endpointServicePair{{
-			endpoint:        "1.1.1.1:11",
-			servicePortName: makeServicePortName("ns1", "ep1", ""),
-		}},
+		expectedNew:   map[proxy.ServicePortName][]*endpointsInfo{},
+		expectedStale: []endpointServicePair{},
```

Comment:
Cases 6 and 8 are non-empty.  I can simplify the net result in another commit, but I really want to leave this code largely unchanged through the refactoring steps.

I made this change in a new commit, and it's actually a nice simplification and removes a bunch of single-use helper functions.